lib/paper_trail.rb

```diff
@@ -126,9 +128,9 @@ def self.paper_trail_store
 
   # Returns PaperTrail's configuration object.
   def self.config
-    @@config ||= PaperTrail::Config.instance
-    yield @@config if block_given?
-    @@config
+    @config ||= PaperTrail::Config.instance
```

Comment:
It was changed from a **class variable** to a **class instance variable**. The change is purely stylistic, and has no effect on thread safety. See https://github.com/airblade/paper_trail/pull/641 for rationale.